utils/scripts/install-ubuntu.sh

```diff
@@ -22,8 +22,10 @@ apt-get install \
     cmake \
     curl \
     fuse3 \
+    gcc \
```

Comment:
I don't think the changes in this file are required, ubuntu isn't used in Jenkins and the tests certainly aren't run on Ubuntu.

Reply:
Just undo the changes there. Thank you!